src/pages/Wishlist/WishlistProducts.jsx

```diff
@@ -0,0 +1,44 @@
+import { Link } from "react-router-dom";
+import { useCart, useWishList } from "../../contexts";
+
+const WishlistProducts=()=>{
+    const {wishlistState,wishlistDispatch} =useWishList();
+    const {wishlist} =wishlistState;
+    const {cartDispatch}=useCart();
+    console.log(wishlist);

```

Comment:
remove console.log

Reply:
yes, I know that, will work on header navigation while refactoring the code.